tests/flashtest.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library         SSHLibrary
+
+*** Test Cases ***
+Open Connection and Log In
+    Open Connection  %{OPI_IP}    timeout=360 seconds
+    Login  root    armbian1234
+    
+Flash firmware
+	# turn APU on and put it in S5 state
+	Write    ./relays.sh ON\n
+	# wait for diodes to turn off
```

Comment:
The diodes are only an indicator, that APU is in S5 state. It would make the test script too complex and I do not really know if there is such a possibility in robot framework. It is much simplier to just wait c.a. 7 seconds and start flashing.